src/plugins/analysis/file_system_metadata/test/test_plugin_file_system_metadata.py

```diff
@@ -52,33 +57,33 @@ def setUp(self):
         self.analysis_plugin = plugin.AnalysisPlugin(self, config=config)
         plugin.FsMetadataDbInterface.__bases__ = (DatabaseMock,)
         plugin.ConnectTo.__enter__ = mock_connect_to_enter
```

Comment:
Why isn't this mocking done with a mock library? Overwriting parameters is generally bad style.

Reply:
changed to `mock.patch`